core/App/components/listItems/ContactListItem.tsx

```diff
@@ -57,9 +57,7 @@ const ContactListItem: React.FC<Props> = ({ contact, navigation }) => {
       <View key={contact.id} style={styles.outerContainer}>
         <View style={styles.textContainer}>
           <Title style={ListItems.contactTitle}>{contact?.alias || contact?.theirLabel}</Title>
-          <Text style={ListItems.contactDate}>
-            {contact.createdAt.toLocaleDateString(i18n.language, dateFormatOptions)}
-          </Text>
+          <Text style={ListItems.contactDate}>{formatTime(contact.createdAt, t)}</Text>
```

Comment:
The way I was thinking about this, is that the date and locale are passed in as parameters (and I guess the format options) everything else is encapsulated by the function.

Reply:
The locale is (kind of) being passed in by the translation function. We need to pass in the translation function because `useTranslations()` is a hook and we can't use hooks outside of components

Reply:
See my comment above. I'm of the opinion that the implementation/function signature should be as agnostic as possible to the npm packages